aries-site/src/pages/foundation/branding.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
-import { Box, Text } from 'grommet';
-import { Hpe } from 'grommet-icons';
+import { Box, Image, Text } from 'grommet';
+import { Aruba, Hpe } from 'grommet-icons';
+import { Button } from 'aries-core';
 
 import {
   ContentSection,
   PageLayout,
   Subsection,
   UsageExample,
 } from '../../layouts';
-import { MainHeading } from '../../components';
+import { SubsectionText } from '../../components';
+
+const ArubaIconExample = () => {
+  const textSize = 'small';
+
+  return (
+    <Box direction="row" align="center" gap="medium">
+      <Aruba color="orange!" size="30px" />
+      <Box direction="row" gap="xsmall">
+        <Text size={textSize} weight="bold">
+          Aruba
+        </Text>
+        <Text size={textSize}>Service Name</Text>
+      </Box>
+    </Box>
+  );
+};
 
 const HpeElementExample = () => {
   const textSize = 'small';
 
   return (
     <Box direction="row" align="center" gap="medium">
-      <Hpe color="brand" size="xlarge" />
+      <Hpe color="brand" size="66px" />
```

Comment:
Do we have to use a customized size instead of t-shirt sizing?

Reply:
Because this is a branding example, the ratio of font-size to icon size (and spacing between icon and text) is important; t-shirt sizing for the does not present the ratios correctly. So, in this case, I believe we need to deviate from t-shirt size for the icon.